userguide/virtualmachines.rst

```diff
@@ -61,7 +61,65 @@ and |ui-add| to open the wizard shown in
 .. figure:: images/virtual-machines-add-wizard-type.png
 
    Add VM
-
+   
+   
+Create a virtual machine by following these seven steps of the creation
```

Comment:
Don't count the number of steps, that will always be out of sync with the actual count.